developer-docs-site/docs/nodes/ait/node-requirements.md

```diff
@@ -43,7 +43,7 @@ Example machine types on various clouds:
 
 ### Implications on hardware requirements
 
-The amount of data stored by the Aptos Blockhain depends on the ledger history (the number of transactions) of the blockchain and the number of on-chain states (e.g., accounts and resources). These values depend on several factors, including: the age of the blockchain, the average transaction rate, and the configuration of the ledger pruner.
+The amount of data stored by the Aptos Blockchain depends on the ledger history (the number of transactions) of the blockchain and the number of on-chain states (e.g., accounts and resources). These values depend on several factors, including: the age of the blockchain, the average transaction rate, and the configuration of the ledger pruner.
```

Comment:
We are no longer title casing this word. Should be "Aptos blockchain". See an initial version of Aptos terms Notion doc: https://www.notion.so/aptoslabs/Aptos-Words-and-Phrases-6b69019dbba24fbdac37f55824dc1639

Reply:
Thanks, Raj! This is now fixed. Please note, this was already in caps and merely misspelled.